### PR TITLE
[WIP] Begin comment integration tests.

### PR DESCRIPTION
**Pull request by jmcarp**
Todo:

* [x] Write basic comment integration test
* [x] Add -parser and fetch a preamble
* [x] Check in fixture cfr changes

[Resolves https://github.com/eregs/notice-and-comment/issues/154]

@cmc333333: to get the proposed test working locally, I parsed a real preamble using -parser as suggested. Unless we want to generate a fixture preamble and cfr changes as part of this, I'm assuming that I'll want to have the integration tests clone -parser and parse a preamble on travis as well. Let me know if that's not right.

### CODE REVIEW

File: regulations/uitests/comment_test.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+from regulations.uitests.base_test import BaseTest
+
+
+class CommentTest(BaseTest, unittest.TestCase):
+
+    job_name = 'Comment test'
+
+    def test_comment(self):
+        self.driver.get(self.test_url + '/preamble/2016_02749')
```
Comment (cmc333333) [inline regulations/uitests/comment_test.py]: Probably want to go to `../I` or similar to better recreate what we expect from users.

**Comment by cmc333333** [comment]
@jmcarp on the money. I think it'd be worth merging as soon as the parser is checked out -- we can follow up with tests around the cfr changes in a later PR.

**Comment by jmcarp** [comment]
Tests passing on sauce labs at https://travis-ci.org/eregs/regulations-site/jobs/124229259. I had to downgrade to python2.7 to run -parser. Also, we should un-hard-code the cfr changes and introduction that I added here once -parser outputs them too.

**Comment by cmc333333** [comment]
Excellent!

**Pull request merged**